lib/devices/ios/ios.js

```diff
@@ -429,6 +429,17 @@ IOS.prototype.setInitialOrientation = function (cb) {
   }
 };
 
+IOS.isSpringBoard = function (uiAppObj) {
+  var isSpringBoard = false;
+  var metaData = uiAppObj['@'];
+
+  if (_.isObject(metaData) && _.has(metaData, "name")) {
```

Comment:
looks like `_.has` already checks that the object is OK for a check: http://underscorejs.org/docs/underscore.html#section-149

so this can just be `if (_.has(metaData, 'name')) {`